shell.middle/src/main/java/org/metaborg/spoofax/shell/commands/SpoofaxEvaluationCommand.java

```diff
@@ -44,8 +44,9 @@ public String description() {
      * @param args
      *            The String to be parsed and executed.
      */
+    @Override
     public void execute(String... args) {
-        onSuccess.accept("Hai, good job.");
-        onError.accept("Oh no..");
+        onSuccess.accept(new StyledText("Hai, good job."));
+        onError.accept(new StyledText("Oh no.."));
```

Comment:
Later this hook will get special spoofax evaluation/parse result classes right, with ASTs and other info? And then the hook will convert it into StyledText so that it can be displayed?